frankenphp.c

```diff
@@ -76,6 +76,8 @@ typedef struct frankenphp_server_context {
   bool finished;
 } frankenphp_server_context;
 
+__thread frankenphp_server_context *local_ctx = NULL;
```

Comment:
nit: why not just `ctx`?

Reply:
I didn't want to shadow other ctx variables while making changes. but I guess at this point, that makes more sense.

Reply:
Turns out renaming it is pretty complex. We sometimes use ctx to refer to the main request as well as the current request. Untangling that is not straightforward. I think it makes sense to do that in a followup pr though.